dot/rpc/modules/rpc.go

```diff
@@ -16,7 +16,25 @@
 
 package modules
 
-import "net/http"
+import (
+	"net/http"
+)
+
+var (
+	// UNSAFE_METHODS is a list of all unsafe rpc methods of https://github.com/w3f/PSPs/blob/master/PSPs/drafts/psp-6.md
+	UNSAFE_METHODS = []string{
```

Comment:
Don't use caps in identifier,  use CamelCase
`UnsafeMethods`

Reply:
done